index.d.ts

```diff
@@ -77,16 +77,18 @@ declare module '@primer/components' {
   type DetailsRenderFunction = (args: {open: boolean}) => React.ReactElement
 
   export interface DetailsProps extends CommonProps, Omit<React.DetailsHTMLAttributes<HTMLDetailsElement>, 'color'> {
-    render?: DetailsRenderFunction
-    children?: DetailsRenderFunction | React.ReactNode
     defaultOpen?: boolean
     overlay?: boolean
-    open?: boolean
     onToggle?: (event: React.SyntheticEvent<HTMLDetailsElement>) => void
-    onClickOutside?: (event: MouseEvent) => void
+    ref?: React.RefObject<HTMLDetailsElement> | null
   }
 
-  export const Details: React.FunctionComponent<DetailsProps>
+  export const Details: React.FunctionComponent<DetailsProps> & {
+    Context: React.Context<{
+      open: boolean | undefined,
```

Comment:
When will `open` be undefined? I _think_ if you use the Details.Context outside of the Details element, React will throw an error. So if you're using Details.Context correctly, open should always be a boolean.

Reply:
I think this was just moved over from the old type definition, not needed!